bellboy/actors/elevator.py

```diff
@@ -16,8 +20,9 @@
 
 def buttonHovered(depth_deque: deque):
     """
-    rough implementation of a buttonHovered method. processes data deque.
-    tells us which button was hovered.
+    rough implementation of a buttonHovered method.
+
```

Comment:
why all this spacing 😞

Reply:
Basically with docstrings, the first line is the 'summary', and anything after that is the comment.

so... standards?

If you'd like everything to be on the same line you can use commas in the future.